### ISSUE TITLE
Get function

### ISSUE THREAD

**Issue by TexRexTex** [cabaletta/baritone#1111]
## Describe your suggestion
get <item> [amount (default 1)]
example 
`get dirt 1`
when this runs it will check for nearby dirt blocks, and if there are none, check chests and even kill players with the block.
`get dragon_egg 1`
will check for any nearby dragon eggs, if found none kill players with it and look in it for chests. it will then check if it is in the end and try and kill the ender dragon. if it is not, in the end, it will search for a nearby ender portal and if it finds none it will get ender 
`get beacon 1` 
will check for beacons, then for nether star and other items needed to craft them if it can't find them it will check for a wither, and if it can't find one it will make one.
get it?

## Settings
noKillWhitelist default empty
killPlayers default false
openChests default true

## Context
This would be one of the most used baritone functions and will be the easiest way to get anything in minecraft.

## Final checklist
- [tick] I have not used any OwO's or UwU's in this issue.

**Comment by TheK0tYaRa** [comment]
Everything but "check players' inventories" seems nice. I doubt the other players' inventories are viewable unless we count as an inventory the slots that are visual, like active slot, offhand or armor.

**Comment by TexRexTex** [comment]
Yes, that's the only thing that might be a problem. 
It would be fine if it is not possible but still a cool idea if it works.

**Comment by ZeroMemes** [comment]
> I doubt the other players' inventories are viewable unless we count as an inventory the slots that are visual, like active slot, offhand or armor.

Correct, the only information supplied to the client by the server in terms of inventory is anything that is visually represented.
